app/src/main/java/com/arctouch/publictransportation/fragment/DepartureListFragment.java

```diff
@@ -0,0 +1,170 @@
+package com.arctouch.publictransportation.fragment;
+
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.StaggeredGridLayoutManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.android.volley.RequestQueue;
+import com.arctouch.publictransportation.R;
+import com.arctouch.publictransportation.domain.Departure;
+import com.arctouch.publictransportation.domain.Route;
+import com.arctouch.publictransportation.domain.generic.params.RouteIdParam;
+import com.arctouch.publictransportation.domain.generic.returns.DeparturesReturn;
+import com.arctouch.publictransportation.interfaces.VolleyListener;
+import com.arctouch.publictransportation.volley.VolleySingleton;
+import com.arctouch.publictransportation.volley.endpoints.DeparturesVolley;
+
+import java.util.List;
+
+public class DepartureListFragment extends Fragment implements VolleyListener<DeparturesReturn> {
+    public static final String ROUTE = "route";
+    public static final String SHOW_ROUTE_DETAILS = "show_route_details";
+
+    private View mRootView;
+    private View mRecyclerViewWeekday;
+    private View mRecyclerViewSaturday;
+    private View mRecyclerViewSunday;
+    private ProgressBar mProgressBar;
+    private TextView mEmptyList;
+
+    private Route mRoute;
+    private boolean mShowRouteDetails;
+
+    public DepartureListFragment() {
```

Comment:
This is unnecessary - if there's no constructor, the compiler will automatically generate an empty one like this, so this can be removed